docs/components/modeler/forms/configuration/forms-config-options.md

```diff
@@ -43,6 +43,10 @@ The expected format for the data is an array of options, each defining its label
 }
 ```
 
+### FEEL expressions
+
+The options are populated from a [FEEL expression](../../feel/language-guide/feel-expressions-introduction.md) that returns an array of options. The resulting array must be formatted as described in the [as above](#input-data-driven-by-process-data).
```

Comment:
"as described in the [as above]" -- what exactly are you referring to here?

Reply:
The "as above" is referring to the example JSON for the input data.

Reply:
Okay -- I would likely reword as "The resulting array must be formatted as described in the [example JSON above]." Does this make more sense?

Reply:
Agreed, I pushed the change 👍